kits/arm/arm.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+import hebi
+from time import sleep, time, perf_counter
+import numpy as np
+
+import os
+import sys
+
+# ------------------------------------------------------------------------------
+# Add the root folder of the repository to the search path for modules
+root_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))
+sys.path = [root_path] + sys.path
+# ------------------------------------------------------------------------------
+
+from util.math_utils import get_grav_comp_efforts
+from util.math_utils import gravity_from_quaternion
+
+class ArmParams(object):
+    
+    def __init__(self, family, moduleNames, hrdf):
+        self.family = family
+        self.hrdf = hrdf
+        self.moduleNames = moduleNames
+
+
+    
+class Arm():
+    
+    def __init__(self, params):
+        # Create arm from lookup
+        self.lookup = hebi.Lookup()
+        sleep(2)
+        print(self.lookup)
+        self.grp = self.lookup.get_group_from_names([params.family], params.moduleNames)
+        print(self.grp)
+        
+        # Create robot model
+        try:
+            self.model = hebi.robot_model.import_from_hrdf(params.hrdf)
```

Comment:
Same as above. This should probably be moved outside of the constructor and into the `create_arm` comment I posted above. An Arm shouldn't be ever created with an invalid hrdf file (_i.e._, we want to prevent "incomplete/invalid" state like this)